minard/views.py

```diff
@@ -565,6 +565,32 @@ def burst():
     data, total, offset, limit = burst_f.load_burst_runs(offset, limit)
     return render_template( 'burst.html', data=data, total=total, offset=offset, limit=limit )
 
+@app.route('/l3')
+def l3():
+    step = request.args.get('step',3,type=int)
+    height = request.args.get('height',20,type=int)
+    data = burst_f.burst_get_cuts()
```

Comment:
this can probably be formatted here or in burst.py, rather than using all the if statements in the html.